src/main/play-doc/developer/BundleConfiguration.md

```diff
@@ -187,7 +222,14 @@ components = {
       "logbroker" = {
         bind-protocol  = "tcp"
         bind-port     = 0
-        services      = ["http://:6379/redis"]
+        service-name  = "redis"
+        acls          = [
+          {
+            tcp = {
```

Comment:
For my understanding, is it correct that we can use `tcp` in the `acls` declaration if we want to declare an `http` endpoint?